### ISSUE TITLE
NVDA doesn't read certain dialog text in Microsoft Word

### ISSUE THREAD

**Issue by jiwi12** [nvaccess/nvda#7510]
NVDA only reads the Ok or Close buttons in certain Word dialogs instead of reading the text automatically. The text can be read with Insert+b as a workaround.

Steps to reproduce:
1. Open a Microsoft Word document that has content, or create one.
2. Go to the Review ribbon with alt+r, then select Word Count... in the Proofing group.

Another example:
1. From within the previous document, go to the Find and Replace dialog with ctrl+h.
2. Type any Find and Replace query and select Replace All.

There are other similar dialogs as well.

Expected behavior:
NVDA should read the text. E.g., "Word has completed its search of the document and has made 10 replacements. Ok button."

Actual behavior:
NVDA only reads the default button in focus unless insert+b is pressed. E.g., "Microsoft Word dialog: Ok button."

System configuration:
NVDA version: 2017.3rc1

NVDA Installed or portable: Installed

Other information:

Windows version: Windows 7 64-bit

Name and version of other software in use when reproducing the issue: Microsoft Word 2007 and 2010

Other questions:

Does the issue still occur after restarting your PC? Yes

Have you tried any other versions of NVDA?
2017.2, same result as above

**Comment by ehollig** [comment]
This works correctly in Word 2016. @bhavyashah can you confirm using Word 2010?

**Comment by bhavyashah** [comment]
This works absolutely correctly on Word 2010 as well. In particular, the Word Count dialog and the one shown after a replacement is issued are read automatically by NVDA, which are the two cases specifically mentioned by @jiwi12.

**Comment by jiwi12** [comment]
Strange. I can reproduce it on two different computers with multiple versions of NVDA and word.

**Comment by ehollig** [comment]
@jiwi12, could you please provide a [Log file](https://github.com/nvaccess/nvda/wiki/LogFilesAndCrashDumps) demonstrating the issue with all addons disabled?

**Comment by jiwi12** [comment]
I just reset NVDA to factory defaults, and it worked correctly in Word.  My preferred configuration had NVDA said not to read tooltips, and apparently the dialogue texts were considered such. Turning tooltips back on fixed the problem. Thanks.

**Comment by jiwi12** [comment]
Sorry, it was actually object descriptions.
